textnow_bot.py

```diff
@@ -44,3 +48,57 @@ def send_message(self, recipient, message):
         self.page.type("#text-input", message)
         self.page.press("#text-input", "Enter")
         self.page.waitForTimeout(500)
+
+
+# Code for Async Below
+
+class AsyncTextNowBot():
+    def __init__(self, page, cookies=None, username=None, password=None):
+        #self.log_in(page, cookies, username, password)
```

Comment:
```suggestion
        # self.log_in(page, cookies, username, password)
```
E265 block comment should start with '# '